TGC/decisions/MEX.txt

```diff
@@ -1354,8 +1354,8 @@ political_decisions = {
 	glory_in_battle_blood_in_the_field = { #madebyFinnBroandAndJPADUN
 		potential = {
 			tag = MEX
-			war_with = USA
-			NOT = { year = 1850 }
```

Comment:
Looks like you picked up some parallel changes again.

Reply:
yea i will merge master in again

Reply:
This still exists

Reply:
resolved